tests/test_utils.py

```diff
@@ -2,9 +2,10 @@
 
 
 def test_constants():
```

Comment:
What is the utility of testing constant values?

Reply:
To be sure that users don't push code with edited constants (for their own tests)